### ISSUE TITLE
Button looks mashed together

### ISSUE THREAD

**Issue by NPadrutt** [SuavePirate/Xamarin.Forms.Controls.FloatingActionButton#14]
I downloaded the sample project and run it on a android emulator. But the button looks like this:

![image](https://user-images.githubusercontent.com/1764367/36895495-5e19745c-1e0f-11e8-8131-70d7ed5417e0.png)

I don't know if that is related, but I had to update the nuget packages (especially the Xamarin Ones) before I could build. 
I had the same in a project of mine. Do you have any idea what that might be?

**Comment by mcferdev** [comment]
Mine was the same... Try install the version 2017.10.17 ! For me, its working fine.

**Comment by SuavePirate** [comment]
Yep it's a bug in the current version from adding the UseCompatPadding flag. Not sure what it's happening but I'm looking into it!

**Comment by mcferdev** [comment]
While don´t have the fix, put the HeightRequest=90 and WidthRequest=80 to Android...To iOS i don´t know if its bugged.

**Comment by SuavePirate** [comment]
iOS does not have this bug, it's just android at the moment

**Comment by mcferdev** [comment]
For who has this problem and need to use while doesn´t have the fix, just put (change the name fabbutton by your name):

```
<fabbutton:FloatingActionButton.WidthRequest>
        <OnPlatform x:TypeArguments="x:Double">
                      <On Platform="iOS">50</On>
                      <On Platform="Android">80</On>
         </OnPlatform>
 </fabbutton:FloatingActionButton.WidthRequest>

 <fabbutton:FloatingActionButton.HeightRequest>
        <OnPlatform x:TypeArguments="x:Double">
                      <On Platform="iOS">50</On>
                      <On Platform="Android">90</On>
         </OnPlatform>
 </fabbutton:FloatingActionButton.HeightRequest>
```

**Comment by ederbond** [comment]
Hey Alex, congratulations for this amazing control you have created.
Do you have an idea of when will this bug gets fixed ?

**Comment by tonholis** [comment]
Related #31

**Comment by sesitosvdx** [comment]
any news? has the bug been solved?
